lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -647,26 +759,25 @@ defmodule ExUnit.DocTest do
     end
   end
 
-  defp inspectable?(<<?#, char, rest::binary>>) when char in ?A..?Z,
-    do: inspectable_end?(rest)
-  defp inspectable?(_),
-    do: false
+  defp inspectable?(<<?#, char, rest::binary>>) when char in ?A..?Z, do: inspectable_end?(rest)
+  defp inspectable?(_), do: false
 
   defp inspectable_end?(<<?., char, rest::binary>>) when char in ?A..?Z,
     do: inspectable_end?(rest)
+
   defp inspectable_end?(<<char, rest::binary>>)
        when char in ?A..?Z
        when char in ?a..?z
        when char in ?0..?9
-       when char == ?_,
-    do: inspectable_end?(rest)
-  defp inspectable_end?(<<?<, _::binary>>),
-    do: true
-  defp inspectable_end?(_),
-    do: false
+       when char == ?_, do: inspectable_end?(rest)
```

Comment:
When you re-run it on master the `, do: ` should go to its own line.

Reply:
merged f989f145b (and did make compile), but no change on that line

Reply:
I ran it locally and it worked. Please try `make clean compile` then just to be double sure.

Reply:
make clean did the trick